www/js/diary/services.js

```diff
@@ -769,38 +774,27 @@ angular.module('emission.main.diary.services', ['emission.plugin.logger',
       }
     }
 
-    timeline.confirmedTrip2Geojson = function(trip) {
+    timeline.compositeTrip2Geojson = function(trip) {
       if (trip == undefined) {
-        return Promise.resolve(undefined);
+        return undefined;
       }
-      Logger.log("About to pull location data for range "
-        + moment.unix(trip.start_ts).toString() + " -> " 
-        + moment.unix(trip.end_ts).toString());
-
-        const fillPromises = [
-            CommHelper.getRawEntries(["analysis/recreated_location"], trip.start_ts, trip.end_ts, "data.ts", 100)
-        ];
```

Comment:
To confirm: We are not calling `CommHelper.getRawEntries` from `confirmedTrip2Geojson` anymore because the returned object from the server already contains a truncated version of its rawEntries, correct?

Reply:
Correct, we don't need a separate request to retrieve the locations from the phone because they are included in the `composite trip` obj

When composite trips are created on the server, it samples 100 raw locations and they are put into the `locations` field. We still have to format the locations into geojson on the phone, though